.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,26 @@
-# What does this PR do?
+**Collection**: [Note which collection this PR will affect]
```

Comment:
I think we still need a header here, or people are going to miss filling out the collection.

Reply:
Ah good point.

Reply:
Added a header.